system/Files/File.php

```diff
@@ -200,7 +200,7 @@ public function getDestination(string $destination, string $delimiter = '_', int
 				{
 					$i = end($parts);
 					array_pop($parts);
-					array_push($parts, ++ $i);
+					$parts[]     = ++ $i;
```

Comment:
Do we have a rector rule that makes the increment/decrement operators close to their subjects?
```diff
- ++ $i;
- $j ++
+ ++$i
+ $j++
```

Reply:
It seems not yet for now.